src/main/java/com/thebois/models/beings/Colony.java

```diff
@@ -42,6 +47,37 @@ private void createBeings(final Iterable<Position> vacantPositions) {
         }
     }
 
+    /**
+     * Allows items to be stored in the shared inventory for the colony.
+     *
+     * @param item The item to be stored
+     */
+    public void addItemToColonyInventory(IItem item) {
```

Comment:
Yeah, will change it. My thought process was that being super clear with what was happening/being done it would be easier to read/understand. Just want to know your thoughts on this. You can resolve this after you seen this.